animations/lib/src/misc/curved_animation.dart

```diff
@@ -0,0 +1,156 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+import 'package:flutter/material.dart';
+
+class CurvedAnimationDemo extends StatefulWidget {
+  static const String routeName = '/misc/curved_animation';
+
+  @override
+  _CurvedAnimationDemoState createState() => _CurvedAnimationDemoState();
+}
+
+class CurveChoice {
+  final Curve curve;
+  final String name;
+  const CurveChoice({this.curve, this.name});
+}
+
+class _CurvedAnimationDemoState extends State<CurvedAnimationDemo>
+    with SingleTickerProviderStateMixin {
+  AnimationController controller;
+  Animation<double> animationRotation;
+  Animation<Offset> animationTranslation;
+  static const _duration = Duration(seconds: 4);
+  List<CurveChoice> curves = [
```

Comment:
Ah! No, I meant you could combine all the choices in both lists to make one bigger one! The `SomethingSomethingOut` curves can also be included here. 😄